src/Style/Input/RadioButtonStyle.ts

```diff
@@ -17,42 +24,60 @@ export const RadioLabel = styled.span`
     left: 0;
     margin-right: 8px;
     border-radius: 50%;
-    width: 1.4em;
-    height: 1.4em;
-    border: ${({ theme }) =>
-      theme === 'white'
-        ? `1px solid ${SecondaryColor.white}`
-        : `1px solid ${SecondaryColor.lightgrey}`};
-    background: ${({ theme }) =>
-      theme === 'white' ? 'transparent' : `${SecondaryColor.white}`};
+    height: 18px;
+    width: 18px;
+    flex-shrink: 0;
+    border: ${({ error, theme }) =>
```

Comment:
Could you help to change to the if-else condition as line 70 - 79? 🙏 

Some pros I can think are:
- it will be scalable for more themes in the future.
- it will be readable and safe to ensure the error color won't be overridden.

Reply:
Agreed!